extensions/wikia/Insights/models/InsightsUnconvertedInfoboxesModel.php

```diff
@@ -43,9 +43,16 @@ public function hasAltAction() {
 	}
 
 	public function getAltActionUrl( Title $title ) {
-		$subpage = Title::newFromText( $title->getText() . "/Draft", NS_TEMPLATE );
+		if ( !class_exists( 'TemplateConverter' ) ) {
+			return '';
+		}
 
-		return $subpage->getFullUrl( [ 'action' => 'edit' ] );
+		$subpage = Title::newFromText( $title->getText() . "/" . wfMessage('templatedraft-subpage')->escaped() , NS_TEMPLATE );
+
+		return $subpage->getFullUrl( [
```

Comment:
We had major issues with `Title::newFromText` returning null if a message key was passed to it due to message rebuild issues. Worth a check for `$subpage === null`?